src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -170,6 +176,36 @@ public class TopicPartitionChannel {
   // Used to identify when to flush (Time, bytes or number of records)
   private final BufferThreshold streamingBufferThreshold;
 
+  // Whether schematization has been enabled.
+  private boolean enableSchematization;
+
+  // Whether schema evolution could be done on this channel
+  private boolean enableSchemaEvolution = false;
```

Comment:
I feel this property should be passed in from SnowflakeSinkServiceV2 and shouldnt be at channel level, instead should be at a table/connector level no? Hence making this private final just makes more sense.